src/main/docker/Dockerfile

```diff
@@ -26,6 +26,15 @@ RUN zypper -n refresh && \
     zypper -n install curl postgresql dejavu-fonts && \
     zypper -n clean --all
 
+# Install gosu
+RUN gpg --batch --keyserver-options http-proxy=${env.HTTP_PROXY} --keyserver hkps://keys.openpgp.org \
+      --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 && \
+    curl -o /usr/local/bin/gosu -SL "https://github.com/tianon/gosu/releases/download/1.10/gosu-amd64" && \
+    curl -o /usr/local/bin/gosu.asc -SL "https://github.com/tianon/gosu/releases/download/1.10/gosu-amd64.asc" && \
```

Comment:
Why v1.10 instead of the latest version?
https://github.com/tianon/gosu/releases

Reply:
I based this on an older example and forgot to check this, updated version to 1.12 now, thanks.